src/core/gltf2importer/bufferparser.cpp

```diff
@@ -106,15 +106,20 @@ bool BufferParser::parse(const QJsonArray &buffersArray, GLTF2ContextPrivate *co
 
 QByteArray BufferParser::dataFromUri(const QString &uri, bool &success) const
```

Comment:
I think that this function would also be useful in the exporter (since we also have to open the buffers there), do you think that it could be set public / static ?